Samples/OIDCMigration/OIDCMigration/SceneDelegate.swift

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright (c) 2022-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+// The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+//
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and limitations under the License.
+//
+
+import UIKit
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
```

Comment:
I think we can safely delete scene delegate, if not using it?

Reply:
I generally agree, but in the case of the sample apps, I like to keep them as close to what Apple provides in their templates. I wouldn't want to imply to developers that the SDK _doesn't_ work with a SceneDelegate, for example.